### PR TITLE
Add TOML syntax highlighting and fix YAML syntax highlighting

### LINKED ISSUE #2761: Add TOML syntax highlighting
That way it will look nicer when crate READMEs have something like this:

image

### PR DESCRIPTION
**Pull request by camelid**
Fixes #2761.

### CODE REVIEW

**Comment by camelid** [comment]
## Before

image

## After

image

File: src/render.rs [hunk 1]
```diff
@@ -40,7 +40,7 @@ impl<'a> MarkdownRenderer<'a> {
                 "language-scss",
                 "language-sql",
                 "language-toml",
-                "yaml",
+                "language-yaml",
```
Comment (camelid) [inline src/render.rs:43]: My local dev environment isn't set up for working on the backend, so I haven't tested this, but I think this is what it should be.

**Comment by camelid** [comment]
Cc @JohnTitor

File: src/render.rs (hunk 1 above)
Comment (Turbo87) [reply src/render.rs:43]: this change is unrelated to "Add TOML syntax highlighting", right?

Comment (camelid) [reply src/render.rs:43]: Correct. I saw this while I was looking at the TOML stuff and I figured I would bundle them into one PR.

Comment (camelid) [reply src/render.rs:43]: Before this change, YAML highlighting wasn't working:

image

And indeed the `language-yaml` class was missing.

**Pull request merged**